internal/sdkv2provider/resource_cloudflare_record_test.go

```diff
@@ -939,6 +962,21 @@ resource "cloudflare_record" "%[2]s" {
 }`, zoneID, rnd)
 }
 
+func testAccCheckCloudflareRecordConfigSVCB(zoneID, rnd string) string {
+	return fmt.Sprintf(`
+resource "cloudflare_record" "%[2]s" {
+	zone_id = "%[1]s"
+	name = "%[2]s"
+	type = "SVCB"
+	data {
+		priority = "2"
+		target   = "foo"
```

Comment:
i had to update the test here to have absolute targets (foo => foo.) since that is what the API returns and we don't want a permadiff here.

Reply:
Thank you!